pkg/api/titond_api.go

```diff
@@ -27,3 +28,33 @@ func (t *TitondAPI) CreateNetwork(data *model.Network) *model.Network {
 
 	return result
 }
+
+func (t *TitondAPI) CreateL2Geth(data *model.Resource) {
+	// TODO : deal with DB
+	t.db.CreateResource()
+
+	namespace := "default"
+	createL2Geth(t.k8s, namespace)
+}
+
+func createL2Geth(client *kubernetes.Kubernetes, namespace string) {
```

Comment:
I think we may use createL2Geth like func (t *TitondAPI) createL2Geth(client *kubernetes.Kubernetes, namespace string).

Reply:
fix it! thank you👍